NuGet.config

```diff
@@ -11,7 +11,7 @@
     <add key="vs-buildservices" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
     <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
     <add key="myget-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
-    <add key="myget-roslyn" value="https://dotnet.myget.org/F/roslyn-analyzers/api/v3/index.json" />
+    <add key="myget-roslyn" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
```

Comment:
(Not) the “key” name doesn’t really make sense anymore since it isn’t Myget. Maybe “dotnet5” would make more sense?

Reply:
I renamed the roslyn one to `dotnet5-roslyn` and the core one to `mygetlegacy-core`.